data-es6.js

```diff
@@ -66,12 +66,10 @@ exports.browsers = {
     short: 'IE 11',
     obsolete: false
   },
-  ie11tp: {
-    full: 'Internet Explorer',
-    short: 'IE<br>Technical<br>Preview',
-    unstable: true,
-    note_id: 'ie-experimental-flag',
-    note_html: 'Have to be enabled via "Experimental Web Platform Features" flag'
+  projectspartan: {
+    full: 'Project Spartan',
```

Comment:
Does it have to be "Project Spartan"? We'll need a shorter name for at least "short" version (fitting stuff in a table is hard). Also, I'd just call test identifier "spartan" (similar to "tr", and not traceur; or "ie11" and not "internet_explorer_11")